api/service/rgen/service.go

```diff
@@ -1,43 +1,43 @@
-package service
+package rgen
```

Comment:
change to randgen or rand_gen is more readable.

Reply:
randgen . done